src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain
+
+
+class Car(
+    val position: Position = Position(POSITION_FIRST),
+) {
+
+    fun moveCar(): Car {
+        return Car(Position(position.value + POSITION_MOVE))
+    }
+
+    companion object {
+        const val POSITION_FIRST = 1
+        const val POSITION_MOVE = 1
```

Comment:
```suggestion
        private const val POSITION_FIRST = 1
        private const val POSITION_MOVE = 1
```
클래스 내부에서만 접근하는 변수라면, private 접근자를 붙여보면 어떨까요? (이하 관련 내용 동일)